core/templates/pages/exploration-player-page/services/content-translation-language.service.ts

```diff
@@ -0,0 +1,136 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to manage the current language being
+ * used for content translations.
+ */
+
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+
+import { ContentTranslationManagerService } from
+  'pages/exploration-player-page/services/content-translation-manager.service';
+import { ExplorationLanguageInfo } from
+  'pages/exploration-player-page/services/audio-translation-language.service';
+import { LanguageUtilService } from 'domain/utilities/language-util.service';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ContentTranslationLanguageService {
+  constructor(
+    private languageUtilService: LanguageUtilService,
+    private contentTranslationManagerService: ContentTranslationManagerService
+  ) {}
+
+  _currentContentLanguageCode: string = null;
+  _explorationLanguageCode: string = null;
+  _languagesInExploration: ExplorationLanguageInfo[] = [];
```

Comment:
Maybe _languageOptions -- I keep reading the name as describing a list of strings.